src/alpine/manifest.json

```diff
@@ -244,7 +244,8 @@
                 "alpine-3.17-local": {
                   "isLocal": true
                 }
-              }
+              },
+              "variant": "v7"
```

Comment:
Looks like this got added in the wrong spot. Should be with the arm32 Dockerfile you added.